packages/node-logger/src/winston.ts

```diff
@@ -45,7 +50,10 @@ export default class HyperDXWinston extends Transport {
 
     const { level, message, meta } = parseWinstonLog(info);
     hdx('Sending log to HyperDX');
-    this.logger.postMessage(level, message, meta);
+    this.logger.postMessage(level, message, {
+      ...meta,
+      ...this.getCustomMeta?.(),
```

Comment:
(same thought for pino) shouldn't we let the log-specific properties overwrite the global trace properties?

Reply:
good point. a debate in my mind before I put this. idk, I was thinking the opposite way. but now when I think again, maybe log level is more important ? 🤷

Reply:
My assumption is that if you're specifying a property at the specific log level, you probably have more particular information to capture compared to the global attribute set earlier at a higher level, for whatever reason. 

I can only think global overwriting local attributes as being desirable if the local attributes were set accidentally, but I assume that's a rare case and one that the user would want to actually fix as well.